### DISCUSSION TITLE
115离线下载失败  failed to add offline download task: invalid character 'd' looking for beginning of value

### DISCUSSION THREAD

**Discussion by Xne3** [AlistGo/alist#7209] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/2bc5b8ae-c2a7-4396-b8e6-0434b9080ad1)![image](https://github.com/user-attachments/assets/46c27a97-e733-47c2-874a-d0698c6b51e9)

之前用的3.36，115挂了之后才更新的3.37.3，第一次用离线下载，但是好像下不了

**Comment by anwen-anyi** [comment, marked as answer]
更新 v3.37.4
 - https://github.com/alist-org/alist/discussions/7211#discussioncomment-10687254

**Comment by Mojovax** [comment]
![ROCUQpdYTF](https://github.com/user-attachments/assets/68e7963a-c6cf-4ebd-b32f-f0905fa55bb5)
我更新到了3.37.4 115的离线下载还是失败

**Reply by anwen-anyi** [reply]
> ![ROCUQpdYTF](https://github.com/user-attachments/assets/68e7963a-c6cf-4ebd-b32f-f0905fa55bb5)
> 
> 我更新到了3.37.4 115的离线下载还是失败

我猜你是Docker，估计没拉到最新的版本

**Reply by Mojovax** [reply]
有可能，我再试试看

**Reply by Mojovax** [reply]
删除镜像，重新拉取了一下，可以了，感谢

**Reply by eiauo** [reply]
3.40.0 115离线下载，报告同样的问题。
magnet:?xt=urn:btih:IVNJMAQER5OOOIQDJJE6OL2XI5BXYGBP

**Reply by Jinshijiming** [reply]
同3.40.0 相同问题

**Reply by Jinshijiming** [reply]
> 同3.40.0 相同问题
测试了3.40-3.41，都会出现这个问题
回退到3.37.4就正常了
